src/ApiService/ApiService/FeatureFlags.cs

```diff
@@ -7,4 +7,5 @@ public static class FeatureFlagConstants {
     public const string EnableCustomMetricTelemetry = "EnableCustomMetricTelemetry";
     public const string EnableBlobRetentionPolicy = "EnableBlobRetentionPolicy";
     public const string EnableDryRunBlobRetention = "EnableDryRunBlobRetention";
+    public const string EnableWorkItemCreation = "EnableWorkItemCreation";
```

Comment:
This include both creation and update, right?

Reply:
That's right, it includes anything to do with work items